gnd/src/main/java/com/google/android/gnd/model/feature/Feature.java

```diff
@@ -52,8 +52,7 @@ public String getTitle() {
   }
 
   public String getSubtitle() {
-    // TODO: Implement subtitle logic.
-    return "";
+    return String.format("Added by %s", getCreated().getUser().getDisplayName());
```

Comment:
This is really ui-specific logic and thus doesn't belong in the model.. could you please move this into the viewmodel instead?

Reply:
Note that arguably "subtitle" is as well, and we'll eventually probably want to move away from having that in the model too.

Reply:
Agree with @gino-m, I wouldn't use `subtitle` to store this.

You could add a `createdBy` field to the ViewModel (which just contains `user.displayName`) and put the "Added by %s" as a string resource for the view (which also makes it available for translation). That avoids creating a dependency on the context.

Reply:
Done